CurrentCard/docs/README.md

```diff
@@ -25,13 +25,13 @@ The high level block diagram of the current sensor card is shown below:
 <img src="images/current-sensor-blockdiagram.svg" width="500%" />
 
 ### Current Sensor
-LEM LA 55-P current sensor is selected for this design, as it is the only sensor available from LEM with an open aperture and PC pins that can measure +/-55A. 
+LEM LA 55-P or LA 100-P current sensors are selected and can be used for this design based on the sensing range desired. They are available from LEM with an open aperture and PC pins. LEM LA 55-P and LA 100-P can measure +/-55A and +/-150A, respectively. Note that the LA100 series also has these variants, LA100-P/SP13 and LA100-TP, which have different gains that users must be careful about when ordering.
```

Comment:
@noguchi-takahiro can you update this line as follows:
Based on the sensing range desired, the current measurement card can use either LA 55-P or LA 100-P from LEM. These sensors have an open aperture and PC pins. The LA 55-P can measure +/-55A while the LA 100-P can measure +/-150A.